packages/appstore/src/modules/traders-hub/account-with-wallets.tsx

```diff
@@ -1,29 +1,100 @@
 import React from 'react';
 import Wallet from 'Components/containers/wallet';
-import { observer } from '@deriv/stores';
-import { useWalletList } from '@deriv/hooks';
-import { Loading } from '@deriv/components';
+import { observer, useStore } from '@deriv/stores';
+import WalletCardsCarousel from 'Components/wallet-cards-carousel';
+import { useContentFlag, useWalletList } from '@deriv/hooks';
+import { Loading, ButtonToggle } from '@deriv/components';
+import { convertWallets } from 'Constants/utils';
+import ButtonToggleLoader from 'Components/pre-loader/button-toggle-loader';
+import { routes } from '@deriv/shared';
+import WalletOptionsAndMultipliersListing from 'Components/wallet-content/wallet-option-multipliers-listing';
+import WalletCFDsListing from 'Components/wallet-content/wallet-cfds-listing';
 
 const AccountWithWallets = observer(() => {
+    const {
+        ui: { is_mobile, is_dark_mode_on },
+        client: { is_landing_company_loaded, active_wallet_loginid, setActiveWalletLoginid },
```

Comment:
In my opinion we have to add `active_wallet_loginid` and `setActiveWalletLoginid` in client-store, because we use it for desktop wallets, for wallets in responsive mode, for wallet modal and we will use it for DTrader header